libs/langgraph/langgraph/utils/pydantic.py

```diff
@@ -1,12 +1,16 @@
 import sys
 import typing
 from dataclasses import is_dataclass
-from typing import Any, Dict, Optional, Union
+from typing import Any, Dict, Optional, Sequence, Union
 
 import typing_extensions
 from pydantic import BaseModel
 from pydantic.v1 import BaseModel as BaseModelV1
 
+# NOTE: this is redefined here separately from langgraph.constants
+# to avoid a circular import
+MISSING = object()
```

Comment:
How could there be a circular import w constants? Does constants import any file?

Reply:
yea, imports `langgraph.types` for backwards compatibility